packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenter/dashboard/add-nsx/dedicatedCloud-datacenter-add-nsx.component.js

```diff
@@ -0,0 +1,17 @@
+import controller from './dedicatedCloud-datacenter-add-nsx.controller';
+import template from './dedicatedCloud-datacenter-add-nsx.html';
+
+export default {
+  bindings: {
+    goBack: '<',
+    serviceName: '<',
+    datacenterId: '<',
+    nsxEdgeCurrentLevel: '<',
+    haveMaximumNsx: '<',
```

Comment:
Most common for a boolean name
```suggestion
    hasMaximumNsx: '<',
```